### PR TITLE
Fixed #15522 -- Added ModelAdmin.delete_queryset().

### PR DESCRIPTION
**Pull request by vangelou**
https://code.djangoproject.com/ticket/15522

### CODE REVIEW

File: docs/releases/1.11.5.txt [hunk 1]
```diff
@@ -4,7 +4,13 @@ Django 1.11.5 release notes
 
 *Under development*
 
-Django 1.11.5 fixes several bugs in 1.11.4.
+Django 1.11.5 adds a minor feature and fixes several bugs in 1.11.4.
```
Comment (timgraham) [inline docs/releases/1.11.5.txt]: Per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), features are not backported to the stable branch. This should target Django 2.0 instead.

File: tests/modeladmin/tests.py [hunk 2]
```diff
@@ -713,3 +716,29 @@ def test_has_module_permission(self):
             self.assertFalse(ma.has_module_permission(request))
         finally:
             ma.opts.app_label = original_app_label
+
+
+@override_settings(ROOT_URLCONF="modeladmin.urls")
```
Comment (timgraham) [inline tests/modeladmin/tests.py]: The correct place for the test looks like `tests/admin_views/tests.py.

**Comment by vangelou** [comment]
I have made the requested changes. Thank you for your comments.

File: tests/admin_views/models.py [hunk 3]
```diff
@@ -957,3 +957,7 @@ def __str__(self):
 
 class RelatedWithUUIDPKModel(models.Model):
     parent = models.ForeignKey(ParentWithUUIDPK, on_delete=models.SET_NULL, null=True, blank=True)
+
+
+class Animal(models.Model):
```
Comment (timgraham) [inline tests/admin_views/models.py]: A new model and modeladmin isn't needed -- use an existing one that's already tested in `test_actions.py`.

File: docs/ref/contrib/admin/index.txt [hunk 4]
```diff
@@ -1351,6 +1351,13 @@ templates used by the :class:`ModelAdmin` views:
     operations. Call ``super().delete_model()`` to delete the object using
     :meth:`.Model.delete`.
 
+.. method:: ModelAdmin.delete_queryset(request, queryset)
+
+    .. versionadded:: 2.0
+
+    The ``delete_queryset`` method is given the ``HttpRequest`` and a ``QuerySet``
+    instance. Overriding this method allows to perform extra actions during the queryset deletion process.
```
Comment (timgraham) [inline docs/ref/contrib/admin/index.txt]: Wrap docs at 79 characters.

**Pull request merged**